File: sorting file/sorting.php
<?php
header('Content-type: text/plain');
$myfile = fopen("vector105.txt", "r") or die("Unable to open file!");
$n = 10000*10;
$v = array();
while (!feof($myfile)) {
    $line = fgets($myfile);
    array_push($v,$line);
}
fclose($myfile);
$start = microtime(true);
sort($v);
$stop = microtime(true)-$start;
echo $stop;

?>

File: sorting file/sorting.py
import time


f = open("vector105.txt","r")
lines = [line.rstrip('\n') for line in f]

a = []
for i in lines:
    a.append(int(i))
start = time.time()
a = sorted(a)
stop = time.time() - start
print(stop)
f.close()